src/drive/web/modules/views/OnlyOffice/Toolbar/HomeLinker.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+
+import { useClient, useQuery, isQueryLoading } from 'cozy-client'
+
+import AppLinker from 'cozy-ui/transpiled/react/AppLinker'
+import { generateUniversalLink } from 'cozy-ui/transpiled/react/AppLinker/native'
+
+import {
+  buildAppsQuery,
+  buildSettingsByIdQuery
+} from 'drive/web/modules/queries'
+
+const HomeLinker = ({ children }) => {
+  const client = useClient()
+  const appsQuery = buildAppsQuery()
+  const contextQuery = buildSettingsByIdQuery('context')
+  const appsResult = useQuery(appsQuery.definition, appsQuery.options)
+  const contextResult = useQuery(contextQuery.definition, contextQuery.options)
+
+  if (isQueryLoading(appsResult) || isQueryLoading(contextResult)) return null
+
+  const apps = appsResult.data
+  const context = contextResult.data
+  const defaultRedirection =
+    context && context.attributes && context.attributes.default_redirection
+
+  let homeApp = null
+  if (!defaultRedirection) {
+    homeApp = apps.find(app => app.slug === 'home')
+  } else {
+    const slugRegexp = /^([^/]+)\/.*/
+    const matches = defaultRedirection.match(slugRegexp)
+    const defaultAppSlug = matches && matches[1]
+    homeApp = apps.find(app => app.slug === defaultAppSlug)
+  }
+
+  const homeSlug = homeApp.slug
```

Comment:
this logic should be mutualized somewhere. We already have this code in the cozy-bar.

Reply:
On pourrait modifier un peu cozy-bar pour isoler cette méthode sans le dispatch et l'exporter afin qu'elle soit disponible dans les apps. Mais je n'arrive pas à mettre à jour la cozy-bar dans Drive pour passer à la v8 :( 

Sinon en attendant je peux sortir ce code dans une méthode et ajouter un TODO en commentaire ?

Reply:
You can add this method to cozy-client and use it here. And add a todo in the cozy-bar to say that the lib should use this new method

Reply:
Je vais la sortir avec un `TODO utiliser celle de Client` et l'ajouter dans Client en parallèle. Je ferai la refacto quand ce sera dispo dans Client